compiler-core/src/parse/snapshots/gleam_core__parse__tests__import_dotted_name.snap

```diff
@@ -0,0 +1,13 @@
+---
+source: compiler-core/src/parse/tests.rs
+expression: "\nimport one.two\n"
+---
+error: Syntax error
+  ┌─ /src/parse/error.gleam:2:12
+  │
+2 │ import one.two
+  │            ^^^ I was not expecting this
+
+This syntax for import is not correct. Probably, you meant:
+- `import one/two` to import module `two` from package `one`
+- `import one.{two}` to import value `two` from module `one`
```

Comment:
I think we could have a shorter message if we can't figure out what the programmer meant, something along the lines of 
"To import a value from a module you have to write it like this: `import one.{Two}`" and then have a link to the relevant piece of the gleam tour for more detail on imports (there's other error messages that do this you could take as reference, like the ones for a wrong use usage)

And then we can provide an hint right under the import if we can figure out a good suggestion that wouldn't result in an error. So all in all it would be something like this:

- If the thing they're trying to import is actually a module value
  ```txt
  error: Syntax error
    ┌─ /src/parse/error.gleam:2:12
    │
  2 │ import one.two
    │        ^^^^^^^ Did you mean `one.{two}`

  This is not a valid import, to import a value from a module you
  have to write: `import one.{two}`.
  
  See: https://tour.gleam.run/basics/modules/
  ```
- If the thing they're trying to import is actually a module type
  ```txt
  error: Syntax error
    ┌─ /src/parse/error.gleam:2:12
    │
  2 │ import one.Two
    │        ^^^^^^^ Did you mean `one.{type Two}`?

  This is not a valid import, to import a type from a module you
  have to write: `import one.{type Two}`.

  See: https://tour.gleam.run/basics/type-imports/
  ```
- If the thing they're trying to use is another module:
  ```txt
  error: Syntax error
    ┌─ /src/parse/error.gleam:2:12
    │
  2 │ import gleam.io
    │        ^^^^^^^^ Did you mean `gleam/io`?

  This is not a valid import, to use the `io` module you have to write
  `import gleam/io`.

  See: https://tour.gleam.run/basics/modules/
  ```
- If it's none of the above then we can have something a bit more generic with
  more options like the one you're displaying here
  
What do you think @lpil?

Reply:
`import one.Two` is not a valid syntax, so I can only produce error messages in the parser. Parser does not have enough info to tell if this type actually exists. So, we can recommend an invalid code. I am not very confident about that.

But it general - I understood your idea and it indeed looks better! I will rework my PR to make errors shorter and provide a link to the docs.

Reply:
Ah right I had missed that this is a syntax error, that makes sense